csrc/fusion_segmenter.h

```diff
@@ -524,13 +524,7 @@ class SegmentCandidateFinder {
       const KernelArgumentHolder& inputs,
       SegmentCandidateFinderOptions options = SegmentCandidateFinderOptions()) {
     auto fusion_copy = std::make_unique<Fusion>(*fusion);
-    if (isDebugDumpEnabled(DebugDumpOption::FusionSegments)) {
-      debug() << "Segment the fusion (Original Fusion Un-modified): "
-              << std::endl;
-      fusion_copy->printMath();
-    }
-    SegmentCandidateFinder scf(std::move(fusion_copy), inputs, options);
-    return std::move(scf.segmented_fusion_);
+    return segment(std::move(fusion_copy), inputs, options);
```

Comment:
The change looks good, but I think the debug dump in the other version of `segment` is incorrectly placed after segmentation. At line 535, the complete fusion is segmented, during which some additional ops may be inserted, so `scf.completeFusion()` may be modified. I think it's supposed to be done before line 535.

Reply:
@wujingyue Could you also fix the print location?

Reply:
Done.